pkg/operator/statefulset.go

```diff
@@ -24,74 +24,154 @@ import (
 
 const configReloaderPort = 8080
 
+type ConfigReloader struct {
+	name           string
+	config         ReloaderConfig
+	reloadURL      url.URL
+	listenLocal    bool
+	localHost      string
+	logFormat      string
+	logLevel       string
+	additionalArgs []string
+	volumeMounts   []v1.VolumeMount
+	shard          int32
```

Comment:
```suggestion
	shard          *int32
```

so we know whether a shard value has been provided or not.